packages/GA4Client/src/common.js

```diff
@@ -7,7 +7,8 @@ var ConsentHandler = require('./consent');
 var EVENT_NAME_MAX_LENGTH = 40;
 var EVENT_ATTRIBUTE_KEY_MAX_LENGTH = 40;
 var EVENT_ATTRIBUTE_VAL_MAX_LENGTH = 100;
-var EVENT_ATTRIBUTE_MAX_NUMBER = 100;
+// maximum event attributes reduced to 99 instead of 100 since now we include send_to as GA4 event parameter
+var EVENT_ATTRIBUTE_MAX_NUMBER = 99;
```

Comment:
I believe that there are some "reserved" key names that don't count towards this.  I know that we don't have a 360 paid account, but I believe the free account has a 25 limit.  Can you try sending 25 event parameters, from a-y, and also `send_to`, so in full sending 26 and see if any get limited?  Or did you already do this, and so make this change?

Reply:
@rmi22186 - you're right, I'll bring it back to 100 and modify the unit testing for it